LICENSE-binary

```diff
@@ -298,6 +298,7 @@ javax.inject:javax.inject:1
 net.java.dev.jna:jna:5.2.0
 net.minidev:accessors-smart:1.2
 org.apache.avro:avro:1.9.2
+org.apache.avro:avro:1.11.3
```

Comment:
should avro:1.9.2 be removed?

Reply:
yes I think so

Reply:
no, actually. There's an unshaded avro in there too related to the AvroFileFormat
```
./share/hadoop/common/lib/avro-1.9.2.jar
./share/hadoop/hdfs/lib/avro-1.9.2.jar
```

I'd like it cut, but for now: it is there

Reply:
alright, thanks for the information